invenio_oauthclient/contrib/orcid.py

```diff
@@ -205,9 +205,14 @@ def account_setup(remote, token, resp):
     with db.session.begin_nested():
         # Retrieve ORCID from response.
         orcid = resp.get('orcid')
-
+        full_name = resp.get('name')
+        
         # Set ORCID in extra_data.
-        token.remote_account.extra_data = {'orcid': orcid}
+        token.remote_account.extra_data = {
+            'orcid': orcid,
+            'full_name': full_name
```

Comment:
For my case I just need access to the `authors full_name` from the `current_user` flask variable. 

The name could also be stored on the user profile (and it might should be added there automatically on `oauth-signup`). 

The only reason I see, right now, for storing the `orcid-full_name` separately on the `extra_data` is that since the user can actually edit his full_name on his invenio-userprofile (after wedding e.g) we might end up with two different names for the same person.